support-frontend/conf/DEV.public.conf

```diff
@@ -17,10 +17,10 @@ identity {
 
   oauth {
     client.id = "0oa53x71jcKEswyiH0x7"
-    issuer.url = "https://profile.thegulocal.com/oauth2/aus3vmj44t2UwrwL40x7"
+    issuer.url = "https://theguardian-dev.oktapreview.com/oauth2/aus3vmj44t2UwrwL40x7"
```

Comment:
I modified  these  as I was getting the below error while running the support site locally 

<img width="622" alt="image" src="https://github.com/guardian/support-frontend/assets/73653255/d4a2a11c-b9ea-4bd5-a2fa-3c991ccf62a2">